airbyte-api/src/main/openapi/config.yaml

```diff
@@ -3778,19 +3778,47 @@ components:
           type: string
     OperatorWebhook:
       type: object
-      required:
-        - executionUrl
       properties:
-        executionUrl:
-          type: string
-          description: The URL to call to execute the webhook operation via POST request.
-        executionBody:
-          type: string
-          description: If populated, this will be sent with the POST request.
         webhookConfigId:
           type: string
           format: uuid
           description: The id of the webhook configs to use from the workspace.
+        webhookType:
+          type: string
+          enum:
+            - generic
+            - dbtCloud
+        generic:
```

Comment:
curious, is the idea to remove generic once the webapp switches over to the dbtCloud one?

Reply:
The idea was actually leave it around, but that's a good point - it's basically dead code, since nothing will be using it.

I'm happy to eliminate it, and just have structured dbtCloud (plus the deprecated fields, which will indeed be removed soon). Also happy to leave it, as a marker to say "generic webhook operations are a possible thing".

Reply:
i think at minimum we should remove the backend code. We can leave the interface around with the above comment so we don't have to redesign it. I have a slight preference to remove, however mainly neutral here.

Reply:
Yeah I'm on board with removing it altogether.